client/me/account-close/main.jsx

```diff
@@ -59,76 +75,102 @@ class AccountSettingsClose extends Component {
+										) }
+									</p>
+									<p className="account-close__body-copy">
+										{ translate(
+											"To close your account, you'll need to {{a}}contact our support team{{/a}}.",
+											{
+												components: {
+													a: <ActionPanelLink href="/help/contact" />,
+												},
+											}
+										) }
+									</p>
+								</Fragment>
+							) }
+						{ ! isLoading &&
+							hasPurchases &&
+							! hasAtomicSites && (
+								<Fragment>
+									<p className="account-close__body-copy">
+										{ translate( 'You still have active purchases on your account.' ) }
+									</p>
+									<p className="account-close__body-copy">
+										{ translate(
+											"To delete your account, you'll need to cancel any active purchases " +
+												'in {{a}}Manage Purchases{{/a}} before proceeding.',
+											{
+												components: {
+													a: <ActionPanelLink href="/me/purchases" />,
+												},
+											}
+										) }
+									</p>
+								</Fragment>
+							) }
+						{ ( isLoading || isDeletePossible ) && (
+							<Fragment>
+								<p className="account-close__body-copy">
 									{ translate(
-										'Account closure {{strong}}cannot{{/strong}} be undone, ' +
-											'and will remove all sites and content.',
-										{
-											components: {
-												strong: <strong />,
-											},
-										}
+										'Account closure cannot be undone and will remove all sites and content.'
 									) }
 								</p>
-								<p>
+								<p className="account-close__body-copy">
 									{ translate(
 										"If you're unsure about what account closure means or have any other questions, " +
-											'please {{a}}chat with someone from our support team{{/a}} before proceeding.',
+											'{{a}}chat with someone from our support team{{/a}} before going ahead.',
 										{
 											components: {
 												a: <ActionPanelLink href="/help/contact" />,
 											},
 										}
 									) }
 								</p>
-							</div>
-						) }
-						{ hasAtomicSites && (
-							<div>
-								<p>
-									{ translate(
-										"To close your account, you'll need to contact our support team. Account closure cannot be undone " +
-											'and will remove all sites and content.'
-									) }
-								</p>
-								<p>
-									{ translate(
-										"If you're unsure about what account closure means or have any other questions, " +
-											"you'll have a chance to chat with someone from our support team before anything happens."
-									) }
-								</p>
-							</div>
+							</Fragment>
 						) }
 					</ActionPanelBody>
 					<ActionPanelFooter>
-						{ ! hasAtomicSites && (
+						{ ( isLoading || isDeletePossible ) && (
 							<Button scary onClick={ this.handleDeleteClick }>
 								<Gridicon icon="trash" />
-								{ translate( 'Close account' ) }
+								{ translate( 'Close Account' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Close Account' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/account-closure/atomic-purchases-messaging)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).